vxfreeswitch/tests/test_voice.py

```diff
@@ -560,12 +561,16 @@ def test_create_call(self):
         msg = self.tx_helper.make_outbound(
             'foobar', '12345', '54321', session_event='new')
 
-        client = yield self.esl_helper.mk_client(self.worker, 'uuid-1234')
-
         with LogCatcher(log_level=logging.WARN) as lc:
             yield self.tx_helper.dispatch_outbound(msg)
         self.assertEqual(lc.messages(), [])
 
+        client = yield self.esl_helper.mk_client(self.worker, 'uuid-1234')
+        # We need to wait for the client to be registered on the worker
+        d = defer.Deferred()
+        reactor.callLater(0.1, d.callback, None)
```

Comment:
That works, although it would be good to also test the unanswered calls
removal too?